vignettes/understand_showcase.Rmd

```diff
@@ -0,0 +1,65 @@
+---
+title: "Understand Showcase"
+output: 
+  html_document:
+    toc: true
+    number_sections: true
+vignette: >
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteIndexEntry{Understand Showcase}
+  %\VignetteEncoding{UTF-8}
+---
+
+
+```{r warning = FALSE, message = FALSE}
+rm(list = ls())
+library(kaiaulu)
+library(visNetwork)
+library(igraph)
+library(data.table)
+```
+
+
+# Parse a sample project folder
```

Comment:
Explain what you are doing in words (outside the code block). What would be in a project folder? Why would you parse? See here for an example notebook https://github.com/sailuh/kaiaulu/blob/7bf8ba6510c4050a44d625965181fb8246b53ad4/vignettes/download_mail.Rmd